Atomic/mount.py

```diff
@@ -449,6 +460,12 @@ def _cleanup_container(self, cinfo):
             except Exception as e:
                 raise MountError(e)
 
+    def _clean_tmp_image(self):
+        # If a temporary image is created with commit,
+        # clean up that too
+        if self.tmp_image is not None:
+            self.client.remove_image(self.tmp_image)
+
```

Comment:
Yes, but in this case some one could theoretically call _clean_tmp_image() a second time.  If they did, you would have a failure, if you reset the value, the second call would not fail.